pilot/pkg/util/network/ip_test.go

```diff
@@ -213,6 +218,15 @@ func TestAllIPv6(t *testing.T) {
 	}
 }
 
+func TestGetPrivateIPs(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	result, flag := GetPrivateIPs(ctx)
```

Comment:
This is not really testing anything? Getting a test coverage number is not the goal, actually making sure the function is returning valid data is

Reply:
got it. removed invalid test cases.